app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,31 @@
+class BookingsController < ApplicationController
+  def show
+    @bookings = Booking.all
```

Comment:
shall we call it index action
and you are showing all bookings, perhaps: 
@bookings = current_user.bookings